PROVISIONING-METAL.md

```diff
@@ -0,0 +1,169 @@
+# Povisioning Bottlerocket on metal
+
+This guide will describe what is needed to properly provision Bottlerocket on bare metal.
+Provisioning Bottlerocket on metal is different than provisioning other general-purpose distros.
+Since Bottlerocket has a `dm-verity`-checked boot and root partition, and is immutable at runtime, we cannot simply provision an image and directly write configuration files.
+Bottlerocket requires a few files to be generated and written to disk at provisioning time in order to boot properly; these files are described below.
+
+> For more information about the hardware that Bottlerocket is currently tested on, see [Hardware](HARDWARE.md).
```

Comment:
I am not sure about the stylistic merits of using a quote like this as a sort of footnote. Maybe just remove the quote marker.